Compilation error on Linux without #include <cstdint>

When compiling this on my Arch Linux box, I get a compilation error:

    /games/Animal-Well-editor/src/image.hpp:10:5: error: ‘uint32_t’ does not name a type
       10 |     uint32_t* data_ = nullptr;
          |     ^~~~~~~~
    /games/Animal-Well-editor/src/image.hpp:7:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
        6 | #include <utility>
      +++ |+#include <cstdint>

As suggested by the compiler, adding this up in the includes *does* fix things up fine:

    #include <cstdint>

(Doing an `#include <stdint.h>` also worked for me, though I assume the suggestion is better.)

I could put this in as a PR instead, but I have no idea if doing it like that would cause compilation programs on Windows or not.  Needs some kind of `#ifdef` to only include that on Linux, maybe?

--- a/src/image.hpp
+++ b/src/image.hpp
@@ -4,6 +4,7 @@
 #include <span>
 #include <string>
 #include <utility>
+#include <cstdint>
 
 class Image {
     uint32_t* data_ = nullptr;

